addenda16_test.go

```diff
@@ -70,14 +70,10 @@ func BenchmarkAddenda16Parse(b *testing.B) {
 func testAddenda16ValidRecordType(t testing.TB) {
 	addenda16 := mockAddenda16()
 	addenda16.recordType = "63"
-	if err := addenda16.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda16.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError

	if addenda16.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda16.recordType)
	}